src/main/java/com/piekoszek/nowaksharedrent/auth/InMemoryAccountRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.piekoszek.nowaksharedrent.auth;
+
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+
+class InMemoryAccountRepository implements AccountRepository {
+    private ConcurrentHashMap<String, Account> inMemoryAccountRepository = new ConcurrentHashMap();
+
+    @Override
+    public void save(Account account) {
+        inMemoryAccountRepository.put(account.getEmail(), account);
+    }
+
+    @Override
+    public boolean existsById(String id) {
```

Comment:
This method could be simplified to 
`return inMemoryAccountRepository.containsKey(id);`